internal/bminventory/inventory.go

```diff
@@ -533,6 +537,13 @@ func (b *bareMetalInventory) RegisterCluster(ctx context.Context, params install
 		}
 	}
 
+	if swag.StringValue(params.NewClusterParams.OpenshiftVersion) != "" {
```

Comment:
it's an enum value that can't be empty, so maybe you should just call for `if !b.versionsHandler.IsOpenshiftVersionSupported(swag.StringValue(params.NewClusterParams.OpenshiftVersion)...`

Reply:
It's not an enum anymore.
If there's a `required` field it still can be empty.
https://swagger.io/docs/specification/data-models/data-types/#string

I can add the minimum length attribute.

Reply:
Right but it's still required parameter so maybe the internal functions should handle it? because the first time i look at the code i was thinking that this value can be empty string.

Reply:
Sure that can work as well.
An empty string won't be found in the hashmap of supported versions.